nginx/hosts

```diff
@@ -0,0 +1 @@
+127.0.0.1   m.thegulocal.com	                # Dotcom
```

Comment:
We already bought `thegulocal.com` domain a while ago. We have not bought `gulocal.com` yet although it is available: